app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Rooms</h1>
+<% @rooms.each do |room| %>
+  <h4><%= room.suite %></h4>
+  <%= "Rate: #{room.rate}" %><br>
+  <%= "hotel: #{room.hotel.name}" %><br>
+  <%= "Number of total guests: #{room.guests.count}" %>
```

Comment:
Be careful here - technically `.count` in this case is an ActiveRecord method that changes the SQL query underneath. So, we want this in the `Room` model inside of a method.